simpl/config.py

```diff
@@ -477,8 +477,15 @@ def parse_cli(self, argv=None, permissive=False):
         options = []
         for option in self._options:
             kwargs = option.kwargs.copy()
-            if kwargs.get('default') is None:
+            original_default = kwargs.get('default')
+            if original_default is not None:
                 kwargs['default'] = argparse.SUPPRESS
+                # Since we're suppressing defaults, we need to somehow preserve
+                # the default value for the help text:
+                help_text = kwargs.get('help', '')
+                kwargs['help'] = '%s (default: %s)' % (
+                    help_text, original_default
+                )
```

Comment:
This changed from

```python
if kwargs.get('default') is None:
    kwargs['default'] = argparse.SUPPRESS
```

to

```python
if kwargs.get('default') is not None:
    kwargs['default'] = argparse.SUPPRESS
    # then add default and help text manually
```

What's going on here? First change I can think of is that default values of `None` will be included in the help/usage output now.